### DISCUSSION TITLE
[氵帖]各位绳匠七夕打算怎么过

### DISCUSSION THREAD

**Discussion by sekiroissekiko** [share121/inter-knot#302] [其他]
如题

**Comment by luoyu007** [comment]
七夕和其他时间唯一不同的就是我要看到别人在那秀恩爱，而我只能打游戏

**Comment by byHydrogen** [comment]
和流萤过

**Comment by ITManCHINA** [comment]
Just something like this...
![49b3608ae18e1a060b52da831fbe234](https://github.com/user-attachments/assets/ce9ea26c-2a6e-4ccf-ae1f-7c9f70df0b3c)

**Reply by kunlinm** [reply]
翻垃圾桶是吧？看看垃圾桶里有没有人家丢掉的项链戒指之类的（手动滑稽）

**Comment by lappslover** [comment]
当然是和绳匠的好帮手邦布一起（其实是跟对象啦）

**Comment by FlameF6** [comment]
骑车

**Comment by dongzhiaidao** [comment]
当然是搞钱啦！暧昧可以让我财富自由吗？可以让我无忧无虑吗？巴卡娜！爱情是我通向公司高层的绊脚石！（斯科特音）

**Comment by IX-11** [comment]
找点有意义的事情，一如既往

**Comment by ShangHuaWenYuan** [comment]
[一❤个❤人❤在❤家❤看❤伊❤埃❤斯❤热❤舞](https://www.bilibili.com/video/BV1HT421k7Rp/?spm_id_from=333.337.search-card.all.click&vd_source=9dbf0038cac01a66dbf38722f8cdaf31)

**Comment by c1iied** [comment]
做一些自己平常不会做的事情

**Comment by lovelolit** [comment]
现在是2024/8/10，3:50.所以是陪你过 ;D

**Comment by wddn123** [comment]
我先解释一下 今天有朋友想玩游戏 我把我的号借给我朋友玩了  白天和晚上上号的不是我本人哈 一堆人约我出去过节呢 大家请不要误会哈 我怎么会惨到在游戏里过节呢哈哈哈哈哈哈哈哈哈哈哈哈哈哈

**Comment by ZhuYuan-148** [comment]
打灰！我要打灰！我要打灰！
